examples/mplot3d/view_planes_3d.py

```diff
@@ -0,0 +1,60 @@
+"""
+======================
+Primary 3D view planes
+======================
+
+This example generates an "unfolded" 3D plot that shows each of the primary 3D
+view planes. The elevation, azimuth, and roll angles required for each view are
+labeled. You could print out this image and fold it into a box where each plane
+forms a side of the box.
+"""
+
+import matplotlib.pyplot as plt
+
+
+def annotate_axes(ax, text, fontsize=18):
+    ax.text(x=0.5, y=0.5, z=0.5, s=text,
+            va="center", ha="center", fontsize=fontsize, color="black")
+
+# (view, (elev, azim, roll))
+views = [('XY',   (90, -90, 0)),
+         ('XZ',    (0, -90, 0)),
+         ('YZ',    (0,   0, 0)),
+         ('-XY', (-90,  90, 0)),
+         ('-XZ',   (0,  90, 0)),
+         ('-YZ',   (0, 180, 0))]
+
+layout = [['XY',  '.',   'L',   '.'],
+          ['XZ', 'YZ', '-XZ', '-YZ'],
+          ['.',   '.', '-XY',   '.']]
+fig, axd = plt.subplot_mosaic(layout, subplot_kw={'projection': '3d'},
+                              figsize=(12, 8.5))
+for i in range(len(axd) - 1):
+    plane = views[i][0]
+    axd[plane].set_xlabel('x')
+    axd[plane].set_ylabel('y')
+    axd[plane].set_zlabel('z')
+    axd[plane].set_proj_type('ortho')
+    axd[plane].view_init(elev=views[i][1][0],
+                         azim=views[i][1][1],
+                         roll=views[i][1][2])
+    axd[plane].set_box_aspect(None, zoom=1.25)
```

Comment:
Would this be a good use for `axs[plane].axis('equal')`? 😄

Reply:
A cutout should work for any aspect box :). But right now that command would only change the data limits! Need https://github.com/matplotlib/matplotlib/pull/23552 to merge in to change the box.

Reply:
Ok, let's to this later then. Since we've already released 3.6 RC1, #23552 can't go into 3.6 anymore.